scanner/src/main/kotlin/experimental/ExperimentalScanner.kt

```diff
@@ -141,15 +142,24 @@ class ExperimentalScanner(
             "Resolved provenance for ${packages.size} packages in $packageProvenanceDuration."
         }
 
+        // Remove all VCS paths from the provenances so that always the full repositories are scanned. The VCS paths
+        // will be used after scanning to correctly filter the scan results.
+        val packageProvenancesWithoutVcsPath = packageProvenances.mapValues { (_, provenance) ->
+            when (provenance) {
+                is RepositoryProvenance -> provenance.copy(vcsInfo = provenance.vcsInfo.copy(path = ""))
```

Comment:
Does this need a case distinction for 'Git-Repo'? (Or are the Git repositories of Git-Repo projects scanned already separately?)

Reply:
Yes, but I don't want to add more special handling for git-repo, instead I have created #5142 as discussed some time ago.

Reply:
Nice!